dashboard/src/main/java/com/google/cloud/tools/opensource/dashboard/DashboardMain.java

```diff
@@ -98,11 +100,38 @@ public static void main(String[] arguments)
       throws IOException, TemplateException, RepositoryException, URISyntaxException,
           ParseException, MavenRepositoryException {
     DashboardArguments dashboardArguments = DashboardArguments.readCommandLine(arguments);
-    Path output =
-        dashboardArguments.hasFile()
-            ? generate(dashboardArguments.getBomFile())
-            : generate(dashboardArguments.getBomCoordinates());
-    System.out.println("Wrote dashboard into " + output.toAbsolutePath());
+
+    if (dashboardArguments.hasVersionlessCoordinates()) {
+      generateAllVersions(dashboardArguments.getVersionlessCoordinates());
+    } else {
+      Path output =
+          dashboardArguments.hasFile()
+              ? generate(dashboardArguments.getBomFile())
+              : generate(dashboardArguments.getBomCoordinates());
+      System.out.println("Wrote dashboard into " + output.toAbsolutePath());
+    }
+  }
+
+  private static void generateAllVersions(String versionlessCoordinates)
+      throws IOException, TemplateException, RepositoryException, URISyntaxException,
+          MavenRepositoryException {
+    List<String> elements = Splitter.on(':').splitToList(versionlessCoordinates);
+    checkArgument(
+        elements.size() == 2,
+        "The version-less coordinates should have one colon character: " + versionlessCoordinates);
+    String groupId = elements.get(0);
+    String artifactId = elements.get(1);
+
+    RepositorySystem repositorySystem = RepositoryUtility.newRepositorySystem();
+    ImmutableList<String> versions =
+        RepositoryUtility.findVersions(repositorySystem, groupId, artifactId);
+    for (String version : versions) {
+      if (version.contains("SNAPSHOT")) {
```

Comment:
This if statement filters locally-installed snapshot. Kokoro Daily job will run DashboardMain for snapshot separately.